recipes/default.rb

```diff
@@ -75,7 +75,18 @@
     group node['ntp']['conf_group']
     mode '0644'
     source 'ntp.leapseconds'
+    source node['ntp']['leapfile_url']
     notifies :restart, "service[#{node['ntp']['service']}]"
+    not_if node['ntp']['leapfile_from_mirror']
+  end
+
+  remote_file node['ntp']['leapfile'] do
+    owner node['ntp']['conf_owner']
+    group node['ntp']['conf_group']
+    mode '0644'
+    source node['ntp']['leapfile_url']
+    notifies :restart, "service[#{node['ntp']['service']}]"
+    only_if node['ntp']['leapfile_from_mirror']
```

Comment:
It'll be better for the logs to just if / else on this attribute. That way you don't have a bunch of extra resources firing and then doing nothing with a not_if